modules/network-observability-packet-translation.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-packet-translation_{context}"]
+= Working with endpoint translation (xlat)
+You can use Network Observability and eBPF to enrich network flows from a Kubernetes service with translated endpoint information, gaining insight into the endpoints serving traffic.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. In the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*.
+. Select *cluster*, and then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource for `PacketTranslation`, for example:
++
+[id="network-observability-flowcollector-configuring-packet-translation_{context}"]
+.Example `FlowCollector` configuration
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  agent:
+    type: eBPF
+    ebpf:
+      features:
+       - PacketTranslation   <1>
+----
+<1> You can start enriching network flows with translated packet information by listing the `PacketTranslation` parameter in the `spec.agent.ebpf.features` specification list.
+ 
+.Example filtering
+When you refresh the *Network Traffic* page you can filter for information about translated packets:
+.. Filter the network flow data based on *Destination kind: Service*.
+.. You can see the *xlat* column, which distinguishes where translated information is displayed, and the following default columns:
+
+* *Xlat Zone ID*
+* *Xlat Src Kubernetes Object* 
+* *Xlat Dst Kubernetes Object*
```

Comment:
There's some funny rendering going on in this area, but I can't tell if it's intended to be part of the previous steps/procedure (in which case it probably needs some use of `--` open blocks and `+` continuations), or if it's supposed to be its own set of ordered steps after the `.Example filtering` (in which case it probably just needs a blank line before the `..`s):

<img width="794" alt="image" src="https://github.com/user-attachments/assets/b3210076-c3cd-49ef-bec9-c591eb589e07" />

Reply:
Ah good catch thank you, indeed I needed to add a space after the first colon and then backspace before "You can manage the display of additional columns" so it doesn't render in a box.